azurerm/internal/services/securitycenter/resource_arm_security_center_contact.go

```diff
@@ -104,7 +105,7 @@ func resourceArmSecurityCenterContactCreateUpdate(d *schema.ResourceData, meta i
 	}
 
 	if d.IsNewResource() {
-		if _, err := client.Create(ctx, name, contact); err != nil {
+		if _, err := azuresdkhacks.CreateSecurityCenterContact(client, ctx, name, contact); err != nil {
```

Comment:
can we add a reference to the issue here so we know when this can be switched back:

```suggestion
		// TODO: switch back when the Swagger/API bug has been fixed: https://github.com/Azure/azure-rest-api-specs/issues/10717 (an undefined 201)
		if _, err := azuresdkhacks.CreateSecurityCenterContact(client, ctx, name, contact); err != nil {
```